synopsys-task/task.json

```diff
@@ -327,6 +393,26 @@
       "visibleRule": "scanType = coverity",
       "groupName": "scanOptions"
     },
+    {
+      "name": "coverityAzureToken",
+      "type": "string",
+      "label": "Azure Token",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "PAT or System.AccessToken has to be passed with legitimate access when Coverity/BlackDuck/Polaris PR comment and BlackDuck FIX PR is enabled.",
```

Comment:
We should only mention about Coverity here since this section is Coverity Specific but Polaris/BlackDuck is also mentioned.

Correct message should be - `PAT or System.AccessToken has to be passed with legitimate access when Coverity pull request comment is enabled.`